src/crimson/os/seastore/random_block_manager/rbm_device.h

```diff
@@ -165,12 +173,28 @@ class RBMDevice : public Device {
   virtual std::string get_device_path() const = 0;
 
   uint64_t get_journal_size() const {
-    return super.journal_size;
+    return shard_info.journal_size;
   }
 
-  static rbm_abs_addr get_journal_start() {
+  static rbm_abs_addr get_journal_start_off() {
```

Comment:
Doesn't look like a meaningful name as there is only 1 superblock in a device. What do you think?